test/versioned/pino/pino.tap.js

```diff
@@ -234,42 +222,18 @@ tap.test('Pino instrumentation', (t) => {
           2,
           'should have log in aggregator after transaction ends'
         )
-        t.same(
-          agent.logs.getEvents(),
-          stringifyLines([childLine, line]),
-          'log should be same in aggregator'
-        )
+
+        agent.logs.getEvents().forEach((logLine, index) => {
+          const formattedLine = logLine()
+          t.validateNrLogLine({ line: formattedLine, message: messages[index], level, config })
+          t.equal(formattedLine['trace.id'], meta['trace.id'], 'should be expected trace.id value')
+          t.equal(formattedLine['span.id'], meta['span.id'], 'should be expected span.id value')
+        })
         t.end()
       })
     })
   })
 
-  t.test('forwarding + local decorating(favor forwarding)', async (t) => {
```

Comment:
We decided that if for some reason a user just has both decoration and forwarding, then sure, we'll do both?